src/Compilers/VisualBasic/Test/Semantic/IOperation/IOperationTests_IBlockStatement_MethodBlocks.vb

```diff
@@ -0,0 +1,198 @@
+
+            VerifyOperationTreeForTest(Of MethodBlockBaseSyntax)(source, expectedOperationTree)
+        End Sub
+
+        <Fact>
+        Public Sub IBlockStatement_PropertyGetBlock()
+            Dim source = "
+Class Program
+    ReadOnly Property Prop As Integer
+        Get'BIND:""Get""
+            If 1 > 2
+            End If
+        End Get
+    End Sub
+End Class"
+
+            Dim expectedOperationTree = "
+IBlockStatement (3 statements, 1 locals) (OperationKind.BlockStatement)
+  Local_1: Prop As System.Int32
+  IIfStatement (OperationKind.IfStatement)
+    Condition: IBinaryOperatorExpression (BinaryOperationKind.IntegerGreaterThan) (OperationKind.BinaryOperatorExpression, Type: System.Boolean, Constant: False)
+        Left: ILiteralExpression (Text: 1) (OperationKind.LiteralExpression, Type: System.Int32, Constant: 1)
+        Right: ILiteralExpression (Text: 2) (OperationKind.LiteralExpression, Type: System.Int32, Constant: 2)
+    IBlockStatement (0 statements) (OperationKind.BlockStatement)
+  ILabelStatement (Label: exit) (OperationKind.LabelStatement)
+  IReturnStatement (OperationKind.ReturnStatement)
+    ILocalReferenceExpression: Prop (OperationKind.LocalReferenceExpression, Type: System.Int32)"
+
+            VerifyOperationTreeForTest(Of MethodBlockBaseSyntax)(source, expectedOperationTree)
+        End Sub
+
+        <Fact>
+        Public Sub IBlockStatement_PropertySetBlock()
+            Dim source = "
+Class Program
+    WriteOnly Property Prop As Integer
+        Set(Value As Integer)'BIND:""Set(Value As Integer)""
+            If 1 > 2
+            End If
+        End Set
+    End Sub
+End Class"
+
+            Dim expectedOperationTree = "
+IBlockStatement (3 statements) (OperationKind.BlockStatement)
+  IIfStatement (OperationKind.IfStatement)
+    Condition: IBinaryOperatorExpression (BinaryOperationKind.IntegerGreaterThan) (OperationKind.BinaryOperatorExpression, Type: System.Boolean, Constant: False)
+        Left: ILiteralExpression (Text: 1) (OperationKind.LiteralExpression, Type: System.Int32, Constant: 1)
+        Right: ILiteralExpression (Text: 2) (OperationKind.LiteralExpression, Type: System.Int32, Constant: 2)
+    IBlockStatement (0 statements) (OperationKind.BlockStatement)
+  ILabelStatement (Label: exit) (OperationKind.LabelStatement)
+  IReturnStatement (OperationKind.ReturnStatement)"
+
+            VerifyOperationTreeForTest(Of MethodBlockBaseSyntax)(source, expectedOperationTree)
+        End Sub
+
+        <Fact>
+        Public Sub IBlockStatement_OperatorBlock()
+            Dim source = "
+Class Program
+    Public Shared Operator +(p As Program, i As Integer) As Integer'BIND:""Public Shared Operator +(p As Program, i As Integer) As Integer""
+        If 1 > 2
+        End If
+
+        Return 0;
+    End Operator
+End Class"
+
+            Dim expectedOperationTree = "
+IBlockStatement (4 statements, 1 locals) (OperationKind.BlockStatement)
+  Local_1:  As System.Int32
```

Comment:
> As System.Int32 [](start = 11, length = 16)

This symbol display doesn't feel right. Perhaps we should output ```<anonymous>``` as a replacement for the name.